packages/api/src/config/brevo-module.config.ts

```diff
@@ -4,5 +4,10 @@ export interface BrevoModuleConfig {
     brevo: {
         apiKey: string;
         BrevoContactAttributes?: BrevoContactAttributesInterface;
+        templateDoubleOptIn: number;
```

Comment:
I would call it templateDoubleOptInId or doubleOptInTemplateId to make it clear that you should pass an id here and not a complete template

Reply:
renamed in [9816120](https://github.com/vivid-planet/comet-brevo-module/pull/7/commits/98161206f9248b68b9efe384162aa0bb6367717f)